osu.Game/Screens/Menu/ButtonSystem.cs

```diff
@@ -286,7 +460,7 @@ private bool onOsuLogo()
                     return true;
 
                 case ButtonSystemState.TopLevel:
-                    buttonsTopLevel.First().TriggerClick();
+                    PlayButton!.TriggerClick();
```

Comment:
Please don't use `!`. Especially after marking the field `CanBeNull`. Check other files for examples of how this should be done, and remove the `nullable disable` rather than adding new jetbrains annotations.